pkg/controllers/resources/nodes/fake_syncer.go

```diff
@@ -240,3 +240,18 @@ func filterOutDaemonSets(pl *corev1.PodList) []corev1.Pod {
 
 	return podsNoDaemonSets
 }
+
+// Filter away physical DaemonSet Pods using annotations to enable scale down
+func filterOutPhysicalDaemonSets(pl *corev1.PodList) []corev1.Pod {
+	var podsNoDaemonSets []corev1.Pod
+
+	for _, item := range pl.Items {
+
+		annotations := item.GetAnnotations()
+
+		if annotations["vcluster.loft.sh/owner-set-kind"] != "DaemonSet" {
```

Comment:
Please check if annotations is nil and please use the constant here (see https://github.com/loft-sh/vcluster/blob/main/pkg/controllers/resources/pods/translate/translator.go)

Reply:
Also I'm assuming to not append any pods to podsNoDaemonSets list if the annotation doesn't exists?

Reply:
Yes exactly thanks for the quick change